spec/routes/content_item_spec.rb

```diff
@@ -11,5 +11,16 @@
           organisation_id: '1'
         )
     end
+
+    it 'routes to #show' do
+      expect(get: organisation_content_item_path(organisation_id: 1, id: 1)).to be_routable
```

Comment:
Doesn't line 16 check that you've added the route in config/routes, and line 17 check that the controller has been setup correctly? That's what I was going for.